static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -315,6 +315,92 @@ const getXaxisPlacementId = (sizes: PrebidSize[]): number => {
     return 13663304;
 };
 
+const getPangaeaPlacementId = (sizes: PrebidSize[]): number => {
+    type PangaeaSection = {
+        sections: Array<string>,
+        lb: number,
+        mmpu: number,
+        dmpu: number,
+    };
+    type PangaeaList = Array<PangaeaSection>;
+    const pangaeaList: PangaeaList = [
```

Comment:
minor: if we are only using `PangaeaList` once, is it worth making a custom type since we are possibly adding an abstraction in for no benefit?

Could do this instead:
`const pangaeaList: Array<PangaeaSection> = [`